release/script_release.py

```diff
@@ -50,7 +50,7 @@ def get_tag_name(version):
 
 
 class ReleaseManager:
-    def __init__(self, release_type, remote_name="canalTP"):
+    def __init__(self, release_type, remote_name="hove-io"):
```

Comment:
Not sure about that one. If someone can confirm it's ok?

Reply:
it's the name of git remote on local machine. if it fails on run time, no worries